Makefile

```diff
@@ -255,22 +263,29 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push OPERATOR_IMAGE=$(BUNDLE_IMG)
 
-.PHONY: fix-csv-replaces
-fix-csv-replaces: $(YQ)
+.PHONY: create-build-file
+create-build-file: $(YQ)
+	$(YQ) -n '.config' > $(BUILD_CONFIG_FILE)
+
+.PHONY: set-authorino-default-image
+set-authorino-default-image: $(YQ)
+	@if [ "$(AUTHORINO_VERSION)" != "latest" ]; then\
+		V="$(DEFAULT_REGISTRY)/$(DEFAULT_ORG)/authorino:$(AUTHORINO_IMAGE_TAG)" $(YQ) eval '.config.authorinoImage = strenv(V)' -i $(BUILD_CONFIG_FILE); \
```

Comment:
I see `.github/workflows/build-images.yaml:57` still refers to the old `authorino_image` file.